packages/portfolio-contract/test/portfolio.exo.test.ts

```
@@ -407,127 +439,160 @@ test('evmHandler deposit handles factoryContract spender', t => {
+        amount: AmountMath.make(USDC, 1_000n),
+        fromChain: 'Arbitrum',
+      },
+      { flowId: 1 },
+      ,
+      {
+        evmDepositDetail: {
+          fromChain: 'Arbitrum',
+          ...permitDetails,
+        },
+      },
+    ],
```

Biome (2.3.13)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)